frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/patient-popup/patient-popup.component.html

```diff
@@ -25,6 +24,23 @@ <h5 class="popover-header">
     <div class="popover-body p-0">
         <table class="table mb-0" style="max-width: 300px">
             <tbody>
+                <tr *ngIf="pretriageFlag$ | async">
+                    <th>Vorsichtungskategorie wählen</th>
+                    <td>
+                        <select
+                            [ngModel]="patientStatus$ | async"
+                            (ngModelChange)="
+                                setPretriageCategory(patient.id, $event)
+                            "
+                        >
+                            <option [value]="'black'">EX</option>
+                            <option [value]="'blue'">SK IV</option>
+                            <option [value]="'red'">SK I</option>
+                            <option [value]="'yellow'">SK II</option>
+                            <option [value]="'green'">SK III</option>
+                        </select>
```

Comment:
Like in #405

Reply:
Maybe disable the select input if `patient.treatmentTime >= this.secondsUntilRealStatus`